dist/tools/pyterm/pyterm

```diff
@@ -106,6 +106,8 @@ class SerCmd(cmd.Cmd):
         self.run_name = run_name
         self.log_dir_name = log_dir_name
         self.newline = newline
+        if formatter:
+            self.fmt_str = formatter
```

Comment:
> Then I have to change the load_config(), too.

well at least you can make a start here. `hasattr()` uses `getattr()` so an exception is thrown but in the end it doesn't make much of a difference, except `hasattr()` looks nicer IMHO.